### PR TITLE
Bug/2549 append or update in action type in notifications

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #2549

### CODE REVIEW

File: test/functionalTest/cases/2549_append_or_update_in_actionType_in_notifications/append_or_update_in_actionType_in_notifications.test [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+# 01. Create subscription for E1/A1
+# 02. Create E1/A1
+# 03. See 'append' in notification and reset
+# 04. Update A1 using PUT /v2/entities/E1/attrs/A1/value
+# 05. See 'update' in notification and reset
+# 06. Update A1 using POST /v2/entities/E1/attrs
+# 07. See 'update' in notification and reset
+# 08. Add A2 using POST /v2/entities/E1/attrs
+# 09. See 'append' in notification
+#
```
Comment (fgalan) [inline test/functionalTest/cases/2549_append_or_update_in_actionType_in_notifications/append_or_update_in_actionType_in_notifications.test]: I'd suggest to add a new step to check that that a "mixed" situation works. In particular:

* Do a `POST /v2/entities/E1/attrs` including both A1 (existing) and A3 (not existing)
* Check that actionType is update for A1 and append for A3

Comment (kzangeli) [inline test/functionalTest/cases/2549_append_or_update_in_actionType_in_notifications/append_or_update_in_actionType_in_notifications.test]: ok

Comment (kzangeli) [inline test/functionalTest/cases/2549_append_or_update_in_actionType_in_notifications/append_or_update_in_actionType_in_notifications.test]: Fixed in 49bbdbd

**Comment by fgalan** [comment]
LGTM

**Pull request merged**